docs/tutorials/pi/make-a-plant-watering-robot.md

```diff
@@ -0,0 +1,483 @@
+---
+title: "How to Make a Plant Watering Robot with a Raspberry Pi"
+linkTitle: "Make a Plant Watering Robot with a Pi"
+weight: 50
+type: "docs"
+description: "Instructions for creating a plant watering robot with a Raspberry Pi single-board computer and Viam."
```

Comment:
```suggestion
description: "Create a plant watering robot with a Raspberry Pi single-board computer."
```

All of our tutorials feature Viam - again, something I want us to stop doing

Reply:
lgtm, done!